R/arx_forecaster_mod.R

```diff
@@ -1,29 +1,83 @@
-arx_epi_forecaster <- function(epi_data, response,
-                               ...,
+#' Direct autoregressive forecaster with covariates
+#'
+#' This is an autoregressive forecasting model for
+#' [epiprocess::epi_df] data. It does "direct" forecasting, meaning
+#' that it estimates a model for a particular target horizon.
+#'
+#'
+#' @param epi_data An `epi_df` object
+#' @param outcome A character (scalar) specifying the outcome (in the
+#'   `epi_df`).
+#' @param predictors A character vector giving column(s) of predictor
+#'   variables.
+#' @param trainer A `{parsnip}` model describing the type of estimation.
+#'   For now, we enforce `mode = "regression"`.
+#' @param args_list A list of customization arguments to determine
+#'   the type of forecasting model. See [arx_args_list()].
```

Comment:
Should users know the default of `arx_args_list()` creates 0, 7, 14 periods lagged predictors and 7 day ahead outcomes?

Reply:
I agree, but I don't want to put this in the docs at the moment. Looking at the function would tell you.